frontend/src/components/interview/components/interview-item/interview-item.tsx

```diff
@@ -45,6 +45,7 @@ const InterviewItem: FC<Props> = ({
     useAppForm<InterviewsUpdateRequestDto>({
       defaultValues: {
         interviewerUserId: interview.interviewer?.id ?? '',
+        status: interview.status ?? '',
```

Comment:
```suggestion
        status: interview.status,
```
interview always has a status